src/main/perl/Process.pm

```diff
@@ -103,6 +103,17 @@ regardless of any value for C<NoAction>.
 By default, commands modify the state and thus C<keeps_state> is
 false.
 
+=item C<sensitive>
+
+A boolean specifying whether the arguments contain sensitive information
+(like passwords). If C<sensitive> is true, the commandline will not be reported
+(by default and when C<log> option is used, the commandline is reported
```

Comment:
Sorry to nitpick but I think "by default when the C<log> option is used" is clearer.